scripts/boot_patch.sh

```diff
@@ -173,7 +173,7 @@ if [ -n "$PREINITDEVICE" ]; then
   echo "PREINITDEVICE=$PREINITDEVICE" >> config
 fi
 [ -n "$SHA1" ] && echo "SHA1=$SHA1" >> config
-RANDOMSEED=$(tr -dc 'a-f0-9' < /dev/urandom | head -c 16)
+RANDOMSEED=$(tr -dc 'a-f0-9' < /dev/urandom 2>/dev/null | head -c 16)
```

Comment:
RANDOMSEED is no longer used and can be deleted

Reply:
Ahh, you're right! This is because of debug_ramdisk being used now?

Reply:
Yes, magiskinit doesn't need random now.